pulsar-client/src/test/java/org/apache/pulsar/client/impl/PulsarClientImplTest.java

```diff
@@ -225,6 +226,20 @@ public void testResourceCleanup() throws PulsarClientException {
         }
     }
 
+    @Test
+    public void testConsumerSchemaNull() throws PulsarClientException {
+        @Cleanup
```

Comment:
This doesn't seem to be part of the PulsarClientImpl test, we should move it to the consumer test.

Reply:
OK. PTAL @RobertIndie